src/OpenLoco/src/Map/TileManager.cpp

```diff
@@ -465,16 +465,52 @@ namespace OpenLoco::World::TileManager
         return height;
     }
 
-    SmallZ getSurfaceCornerHeight(const SurfaceElement& surface)
+    static uint8_t getCornerDownMask(const uint8_t cornerUp)
+    {
+        switch (cornerUp)
+        {
+            case SurfaceSlope::CornerUp::north:
+                return SurfaceSlope::CornerDown::south;
+
+            case SurfaceSlope::CornerUp::south:
+                return SurfaceSlope::CornerDown::north;
+
+            case SurfaceSlope::CornerUp::west:
+                return SurfaceSlope::CornerDown::east;
+
+            case SurfaceSlope::CornerUp::east:
+                return SurfaceSlope::CornerDown::west;
+        }
+        return SurfaceSlope::flat;
+    }
+
+    // TODO: rename this function
```

Comment:
Could you rename this? Is this even used by anything other than the mountain tool? Maybe it doesn't make sense for it to be here.

Reply:
It's not used by anything but the mountain tool as of yet. Had trouble coming up with a more appropriate name, though. `getSurfaceCornerDownHeight`?

Reply:
sure, anything to avoid it confusing users who will tend to be using the normal getSurfaceCornerHeight